### PR TITLE
*: improve pre-commit hook

### PR DESCRIPTION
**Pull request by zhexuany**
1. commented make test
2. improved goword test

### CODE REVIEW

**Comment by zhexuany** [comment]
@coocood @hanfei1991 PTAL.

File: hooks/pre-commit [hunk 1]
```diff
@@ -13,15 +13,16 @@ readonly green=$(tput bold; tput setaf 2)
...
-echo -ne "Checking that it builds..."
-if ! OUT=$(make 2>&1); then
-    echo
-    echo "${red}${OUT}"
-    exit_code=1
-else
-    echo "${green}OK"
-fi
-echo "${reset}"
+# comment it by default. You can uncomment it.
```
Comment (shenli) [inline hooks/pre-commit]: Why comment this?

Comment (zhexuany) [inline hooks/pre-commit]: @coocood complains about building is too slow. And sometime, we have to commit even our changes break build.

**Comment by shenli** [comment]
LGTM

**Pull request merged**